ru/game.php

```diff
@@ -116,4 +116,55 @@
     'preset' => 'Правила',
     'save' => 'Сохранить',
     'translationBy' => 'metallicLurker', //Contact me on any uncertainties or details. -mLurker
+	
+	//New strings from July 29th -mLurker
+	'rescaleNow' => 'Обновлять поле других игроков сразу',
+	'rescaleNowInfo' => 'Если не отметить, то поля новых игроков появятся на след. раунде.',
+	'DAScancel' => 'Освежать ЗАП', //As in, "Refresh DAS". -mLurker
+	'DAScancelInfo' => 'Если отметить, то Авто-Передвижение будет отменяться со сменой сторон передвижения.',
+	'FPSDAS' => 'Считать ЗАП по кадрам',
+	'FPSDASInfo' => 'Использует мониторные кадры (60 FPS) для считывания ЗАП. Точность cчёта времени будет снижена.',
+	'webGL' => 'Использовать WebGL',
+	'KPP' => 'Наж./Ф.',
+	'editRoom' => 'Изменить',
+	'savePreset' => 'Сохранить настройки',
+	'overflowRooms' => 'Запасные комнаты', //As in, "Extra Rooms". -mLurker
+	'spectateRooms' => 'Комнаты наблюдения',
+	'gmodeTeam' => 'Командн.',
+	'applyCh' => 'Применить изменения',
+	'audioCredits' => 'Авторы аудиофайлов:', //As in, "Authors of the audiofiles". -mLurker
+	'fullScreen' => 'В полный экран',
+	'showHoldQueue' => 'Показать З.+Оч.',
+	'showStats' => 'Показать статистику',
+	'roomLimits' => 'Ограничения на вход в комнату',
+	'setLimitsInfo' => 'Вы можете поставить ограничения на вход в вашу комнату. Оставьте поля пустыми для снятия ограничении.',
+	'addLimit' => 'Добавить ограничения',
+	's40L' => 'Время Спринта 40L',
+	'accGametime' => 'Общ. время игры',
+	'applyClose' => 'Применить и закрыть',
+	'teamSel' => 'Выбор команды',
+	'myTeam' => 'Вы играете за',
+	'seconds' => 'секунды',
+	'minutes' => 'минуты',
+	'hours' => 'часы',
+	'ms' => 'мс',
+	's' => 'с',
+	'hrs' => 'ч',
+	'attackTable' => 'Таблица атаки',
+	'comboTable' => 'Таблица комбо-атаки',
+	'gBlocking' => 'Блокировка мусора',
+	'blocksType' => 'Фигуры',
+	'randomizer' => 'Генератор случ. фигур', //As in, "Random Piece Generator". No short Russian term for "Randomizer" found. -mLurker
+	'previews' => 'Длина очереди', //As in, "Next queue length". -mLurker
+	'solidGarbage' => 'Заполнение полными линиями', //As in, "Fill with solid lines". -mLurker
+	'solidAfterSecs' => 'После',
+	'lockDelay' => 'Задержка фиксации',
+	'clearDelay' => 'Задержка очистки',
+	'speedLimit' => 'Лимит скорости',
+	'gravityLvl' => 'Гравитация',
+	'garbageDelay' => 'Задержка мусора',
+	'garbageMess' => 'Замешанность мусора',
+	'solidAtk' => 'Атака полными линиями',
+	'noFW' => 'Запрет 4-wide',
+	'hostStart' => 'Начало раунда с хозяина', //As in, "Begin round from the host". -mLurker
```

Comment:
Is shorter version possible? About 13 chars max.
All these room options are in Create room->Advanced, and there is not much space.
If not possible, it's fine, we can try